.travis.yml

```diff
@@ -1,5 +1,27 @@
-rvm:
-  - "2.3.1"
-script: "./test_all_impls.sh"
 language: ruby
+
 sudo: false
+
+env:
+  global:
+    - SASS_LIBSASS_PATH=$TRAVIS_BUILD_DIR/../libsass
+
+matrix:
+  fast_finish: true
+  include:
+    - gemfile: Gemfile
+      env: LANGUAGE_VERSION=3.4
+    - gemfile: Gemfile.sass_3_5
+      env: LANGUAGE_VERSION=3.5
+    - gemfile: Gemfile.sass_4_0
+      env: LANGUAGE_VERSION=4.0
+
+before_install:
+  - rm Gemfile.lock
+  - git clone https://github.com/sass/libsass.git $TRAVIS_BUILD_DIR/../libsass
```

Comment:
OK, I guess you're thinking about pinning to 3.3.6 then? Anything a release happens, then this travis.yml gets updated? I guess later it can do what Ruby Sass does and try Stable + git.